furusiya/function_mess.py

```diff
@@ -39,6 +39,17 @@ def msgbox(text, width=50):
 
 
 def handle_keys():
+    if (Game.player.turns_to_rest > 0
+            and not [
+                e
+                for e in Game.area_map.entities
+                if e.hostile and (e.x, e.y) in Game.renderer.visible_tiles
+            ]):
+        Game.player.turns_to_rest -= 1
+        Game.player.rest()
+        Game.turn = None
```

Comment:
This makes me cringe. Who's turn is `None`? It's not obvious. (Scrollin down, it's the other AIs turns...)